utils/loggers/__init__.py

```diff
@@ -37,14 +47,32 @@
 try:
     import clearml
 
-    assert hasattr(clearml, '__version__')  # verify package import not local dir
+    assert hasattr(clearml, "__version__")  # verify package import not local dir
 except (ImportError, AssertionError):
     clearml = None
 
+try:
+    assert RANK in {0, -1}
```

Comment:
Let's not use `assert` as it can be disabled, let's check the value with:
```
if RANK not in [0, -1]:
  comet_ml = None
else:
  import comet_ml
```